src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -245,6 +220,17 @@ void CascadiaSettings::_ValidateProfilesHaveGuid()
     }
 }
 
+// Method Description:
+// - Resolves the "defaultProfile", which can be a profile name, to a GUID
+//   and stores it back to the globals.
+void CascadiaSettings::_ResolveDefaultProfile()
+{
+    const auto unparsedDefaultProfile{ GlobalSettings().GetUnparsedDefaultProfile() };
+    auto maybeParsedDefaultProfile{ _GetProfileGuidByName(unparsedDefaultProfile) };
+    auto defaultProfileGuid{ Utils::CoalesceOptionals(maybeParsedDefaultProfile, GUID{}) };

```

Comment:
If the user didn't supply a `defaultProfile` before, was the default value `GUID{}` or `std::nullopt`?

Reply:
It was "a GUID default-initialized by the GlobalAppSettings constructor" :smile:

Reply:
(so `{}`)